spec/design/skel/TCB_H.thy

```diff
@@ -47,7 +47,7 @@ fun tcbEPFindIndex where
 
 declare tcbEPFindIndex.simps[simp del]
 
-#INCLUDE_HASKELL SEL4/Object/TCB.lhs Arch= bodies_only NOT liftFnMaybe assertDerived archThreadGet archThreadSet asUser sanitiseRegister getSanitiseRegisterInfo takeWhileM sort_key tcbEPFindIndex
+#INCLUDE_HASKELL SEL4/Object/TCB.lhs Arch= bodies_only NOT liftFnMaybe assertDerived archThreadGet archThreadSet asUser sanitiseRegister getSanitiseRegisterInfo takeWhileM sort_key tcbEPFindIndex awaken
```

Comment:
It might be possible to let it include `awaken` as well (see below). The fewer special cases the better..